gp-ecommerce-fields/gpecf-deduct-deposit.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Gravity Perks // GP eCommerce Fields // Deduct Deposit from Order Summary
+ * https://gravitywiz.com/documentation/gravity-forms-ecommerce-fields/
+ *
+ * Plugin Name:   GP eCommerce Fields - Deduct Deposit from Order Summary
+ * Plugin URI:   https://gravitywiz.com/documentation/gravity-forms-ecommerce-fields/
+ * Description:  This snippet uses a Produdct field to create a deposit field and deducts the deposit from Order Summary
+ * Author:       Gravity Wiz
+ * Version:      0.1
+ * Author URI:   https://gravitywiz.com
+ */
+add_action( 'wp_loaded', function() {
+
+	if( ! function_exists( 'gp_ecommerce_fields' ) ) {
+		return;
+	}
+
+	remove_action( 'gform_product_info', array( gp_ecommerce_fields(), 'add_ecommerce_fields_to_order' ), 9, 3 );
+	add_action( 'gform_product_info', function( $order, $form, $entry ) {
+
+		// CHANGE: Update "123" to the ID of your form.
+		if( $form['id'] != 123 ) {
+			return gp_ecommerce_fields()->add_ecommerce_fields_to_order( $order, $form, $entry );
+		}
+
+		// CHANGE: Update the "2" to your deposit field ID.
```

Comment:
Update this comment to read:
> // Update the "2" to your deposit field ID.

Reply:
Done, but I was wondering if this should be 4 instead of 2 per the SL Code organization formating documentation.

Reply:
Great catch! Yes, let's go ahead and update that.